internal/manifest/v1beta1/spec_resolver.go

```diff
@@ -83,7 +83,7 @@ func (m *ManifestSpecResolver) Spec(ctx context.Context, obj declarative.Object)
 	case v1beta1.HelmChartType:
 		mode = declarative.RenderModeHelm
 	case v1beta1.OciRefType:
-		mode = declarative.RenderModeHelm
+		mode = declarative.RenderModeRaw
```

Comment:
The previous oci layer was rendered as helm chart, but currently changed to Raw manifest, it will introduce breaking changes when we handle those helm chart oci layer.

Reply:
True. I have prepared [this](https://github.com/kyma-project/lifecycle-manager/pull/469) CLI change to stop using helm-chart bundling in OCI layers created for the module templates.

Reply:
The thing is you need to communicate to other teams that we introduced these breaking changes because they already rendered module template with helm chart oci layer.